src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -143,31 +160,107 @@ def apply_average_tropomi_operator(
 
         # If building Jacobian matrix from GEOS-Chem perturbation simulation sensitivity data:
         if build_jacobian:
-            # Get GEOS-Chem perturbation sensitivities at this lat/lon, for all vertical levels and state vector elements
-            sensi_lonlat = GEOSCHEM["Sensitivities"][
-                gridcell_dict["iGC"], gridcell_dict["jGC"], :, :
-            ]
-            # Map the sensitivities to TROPOMI pressure levels
-            sat_deltaCH4 = remap_sensitivities(
-                sensi_lonlat,
-                merged["data_type"],
-                merged["p_merge"],
-                merged["edge_index"],
-                merged["first_gc_edge"],
-            )  # mixing ratio, unitless
-            # Tile the TROPOMI averaging kernel
-            avkern_tiled = np.transpose(np.tile(avkern, (n_elements, 1)))
-            # Tile the TROPOMI dry air subcolumns
-            dry_air_subcolumns_tiled = np.transpose(
-                np.tile(dry_air_subcolumns, (n_elements, 1))
-            )  # mol m-2
-            # Derive the change in column-averaged XCH4 that TROPOMI would see over this ground cell
-            jacobian_K[i, :] = np.sum(
-                avkern_tiled * sat_deltaCH4 * dry_air_subcolumns_tiled, 0
-            ) / sum(
-                dry_air_subcolumns
-            )  # mixing ratio, unitless
+            
+            # redundant code here...
```

Comment:
Can you put a comment here with a TODO to remove the redundancy between this and the average tropomi operator? I assume this is what the comment is referring to?

Reply:
Added comment in https://github.com/geoschem/integrated_methane_inversion/pull/252/commits/e365983852f1aa6aae31db51c8d9fb2eaa7937a5